docs/schemas/scenario/jobs/qgis-installation-finder.json

```diff
@@ -3,9 +3,8 @@
   "$id": "https://github.com/qgis-deployment/qgis-deployment-toolbelt-cli/raw/main/docs/schemas/scenario/jobs/qgis-installation-finder.json",
   "description": "Job in charge of findind installed QGIS version on computer.",
   "title": "QGIS installation finder",
-  "type": "array",
+  "type": "object",
```

Comment:
I've worked in the same time on this but I've more changes. https://github.com/qgis-deployment/qgis-deployment-toolbelt-cli/compare/main...docs/json-schema-qgis-finder